### PR TITLE
Miscellaneous Documentation Fixes Jan16

### PR DESCRIPTION
**Pull request by rglazer2**
Addresses #4062, #3381, #3909, #4257

### CODE REVIEW

**Comment by rglazer2** [comment]
@mjwitte How does this all look? I think I still need an example report for Component Cost Economics Summary in OutputDetailsAndExamples.md, but I wasn't sure how to get that.

File: doc/src/docs/InputOutputReference/06-StandardOutputReports.md [hunk 1]
```diff
@@ -510,6 +510,18 @@ The Standard 62.1 Summary (key: Standard62.1Summary) produces a report that is c
...
+
+The LEED Summary report provides many of the simulation results required for certification of Energy and Atmosphere Credit 1 Optimized Energy Performance according to the LEED Green Building Rating System™. The report can be produced by specifying LEEDSummary in Output:Table:SummaryReports which is also part of the AllSummary option. Directly following is an example of this report.
+
+#### Object Count Summary
+
+The Object Count Summary provides the count on the number of specific objects in the file. The key used to obtain this report is ObjectCountSummary.
+
+#### Component Cost Economics Summary
+
+The Component Cost Economics Summary provides the construction cost estimate summary for the project. The costs are broken into eight catagories and the reference building costs are provided as a comparison. A second table is also produced that provides line item details with one line for every line item object.
```
Comment (mjwitte) [inline doc/src/docs/InputOutputReference/06-StandardOutputReports.md]: Please add the report key for this one.  Otherwise this section looks good.

**Comment by mjwitte** [comment]
Example file 5ZoneCostEst.idf should produce the component cost summary.

**Comment by rglazer2** [comment]
@mjwitte Let me know if there is anything else I need to do for this.

**Comment by mjwitte** [comment]
I think it's good, except I can't get the new component cost table to render.  Looks like there may be issues elsewhere in the md file.  Or it may just be my md tools.

**Comment by rglazer2** [comment]
@mjwitte Everything looks fine when I viewed it with MarkdownPad 2

**Comment by mjwitte** [comment]
Doesn't load completely here in stackedit, but can view in MarkDownPad 2.  @Myoldmopar this is ready for review.

**Comment by Myoldmopar** [comment]
We haven't had too many image diffs, so in case you haven't seen them, check out the diff and play with the different options [here](https://github.com/NREL/EnergyPlus/pull/5402/files#diff-5e8424d4b330bc4fefe90160269056c9).

**Comment by nrel-bot-2** [comment]
@rglazer2 @lgentile it has been 14 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
@mjwitte I was able to get the raw html table to render.  I'm going to merge this in.  I'll make a note to myself to check the output of the rendered documentation during test builds.

**Pull request merged**